eventespresso/event-espresso-core
# Added button to Transaction admin page for recalculating line items. Closes #726

## Problem this Pull Request solves
plz see #726
After opening up the appropriate files and looking into the code a bit I realized that all that was needed was to copy one of the existing buttons on the transaction admin details page, then add a new route and callback for calling `EEH_Line_Item::apply_taxes()` on the Transactions total line item. This was literally only about 15 minutes of work on top of the 5-10 minutes it took to open the files and search through all of the code to determine what needed to be done. I concluded it would be more efficient to simply add the few lines of code required to do this while I already had everything open.

## testing

- log into the admin, go to the Transactions admin page, and click on any Transaction to view its details page
- [x] confirm that a new button labeled "Recalculate Taxes and Total" appears below the **Transaction Items** table
- take note of the Transaction's tax line items (or lack thereof) and grand total
- in the first column of the **Transaction Items** table, click on the Event link for one of the tickets so that the Event editor opens in another tab or window
- find the ticket in the editor and click to view its advanced settings
- click on the "This ticket is taxable" checkbox to change whatever it is currently set at and click **Save**
- return to the Transaction admin details page in the other tab or window and click on the "Recalculate Taxes and Total" button
- [x] confirm that the taxes and transaction total have been updated as expected
- experiment with taxes and transactions in this branch and attempt to find any potential issues
- plz compare any issues against master branch to determine if they are caused by the work in this branch
